mozilla_aws_cli/login.py

```diff
@@ -331,6 +311,7 @@ def validate_id_token(self):
             logger.error('ID Token failed validation : {}'.format(e))
             return None
         logger.debug("ID token dict : {}".format(self.id_token_dict))
+        return self.id_token_dict
 
 
     def get_role_map(self):
```

Comment:
Since this has the potential to now be called multiple times, should it have a check at the beginning to return `self.role_map` if it is not None?

Reply:
I checked and there is no code path that could call this multiple times